ui/admin/app/routes/scopes/scope/roles/role/scopes.js

```diff
@@ -0,0 +1,20 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { GRANT_SCOPE_KEYWORDS } from 'api/models/role';
+
+export default class ScopesScopeRolesRoleScopesRoute extends Route {
+  // =services
+
+  @service store;
+
+  async model() {
+    const role = this.modelFor('scopes.scope.roles.role');
+    if (!GRANT_SCOPE_KEYWORDS.some((id) => role.grant_scope_ids.includes(id))) {
+      await this.store.query('scope', {
+        scope_id: role.scope.id,
+        recursive: true,
```

Comment:
should this have `pushToStore: false` in case role scope is global and it recursively loads a lot of scopes?